app/src/main/res/values-cs/ 	strings.xml

```diff
@@ -0,0 +1,65 @@
+<resources
+    xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+
+    <string name="app_name">Loyalty Card Locker</string>
+    <string name="action_add">Přidat</string>
+
+    <string name="noGiftCards">ZAtím némáte žádné věrnostní karty. Klikněte na tlačítko "+" (plus) nahoře a začněte.\n\nLoyalty Card Locker umožňuje nosit své věrnostní karty v telefonu, takže jsou vždy na dosah.</string>
+
+    <string name="storeName">Obchod</string>
+    <string name="note">Poznámka</string>
+    <string name="cardId">ID karty</string>
+    <string name="barcodeType">Typ čárového kódu</string>
+
+    <string name="cancel">Zrušit</string>
+    <string name="save">Uložit</string>
+    <string name="capture">Naskenovat kartu</string>
+    <string name="enterCard">Vložit vlastnoručně</string>
+    <string name="edit">Editovat</string>
+    <string name="delete">Smazat</string>
+    <string name="confirm">Potvrdit</string>
+    <string name="deleteTitle">Odstzranit věrnostní kartu</string>
+    <string name="deleteConfirmation">Opravdu chcete smazat tuto věrnostní kartu?</string>
+    <string name="ok">Ano</string>
+    <string name="copy_to_clipboard">Kopírovat ID do schránky</string>
+
+    <string name="editCardTitle">Editovat věrnostní kartu</string>
+    <string name="addCardTitle">Přidat věrnostní kartu</string>
+    <string name="viewCardTitle">Zobrazit věrnostní kartu</string>
+    <string name="scanCardBarcode">Oskenujte kód karty</string>
+
+    <string name="barcodeImageDescription">Obrázek kódu karty</string>
+
+    <string name="noStoreError">Nebyl zadán Obchod</string>
+    <string name="noCardIdError">Nebylo zadáno ID karty</string>
+
+    <string name="cardIdFormat">%1$s: %2$s</string>
+    <string name="storeNameAndNoteFormat" translatable="false">%1$s - %2$s</string>
```

Comment:
You can remove this line from the translations, as it is not necessary. This string is marked as not translatable